### DISCUSSION TITLE
Introduction and questions regarding the focus of this project's GSoC

### DISCUSSION THREAD

**Discussion by opnuub** [unicode-org/lstm_word_segmentation#39] [General]
Dear Unicode team,

I am Shenghong Liu, currently studying Electronic and Information Engineering at Imperial College London. I love languages and understand Japanese, Cantonese and Mandarin writing. I also have experience developing ML models on Computer Vision, Time Series Forecasting and Natural Language Processing using TensorFlow and PyTorch. In 2023, I attained the TensorFlow Developer Certificate. Hence, I'm delighted to have found this project and would love to contribute to this project for GSoC 2025. 

I saw that a lot of importance is placed on enhancing the current model. Hence, I'll like to ask about the weightage/importance of other objectives (in the proposal) such as building a reproducible test suite for 3 languages and training a model with higher accuracy in a major language and a DDL. Thank you very much for your time.

Yours sincerely,
Shenghong Liu

**Comment by sffc** [comment]
A compelling proposal will suggest an approach that is likely to improve upon the performance of the current model while still being likely to retain high accuracy and small data size. In order to instill confidence that your new model can be used in place of the current model, we also need to see a comparison of how it performs on real-world test cases and how it scales to DDLs.
